functions/src/api/temples/index.ts

```diff
@@ -38,6 +38,7 @@ templesRouter.post('/', validator({body: CreateTempleData}), async ctx => {
     index: 0,
     playing: false,
     contentId,
+    facilitator: ctx.user.id,
```

Comment:
I would like some kind of middleware like the Koa middleware, that requires an authorized request here. Basically making sure that `ctx.user` is set